bigbluebutton-html5/imports/ui/components/actions-bar/service.js

```diff
@@ -8,13 +9,24 @@ const getBreakouts = () => Breakouts.find({ parentMeetingId: Auth.meetingID })
   .fetch()
   .sort((a, b) => a.sequence - b.sequence);
 
-const getUsersNotAssigned = (users) => {
-  const breakouts = getBreakouts();
-  const breakoutUsers = breakouts
-    .reduce((acc, value) => [...acc, ...value.users], [])
-    .map(u => u.userId);
-  return users.filter(u => !breakoutUsers.includes(u.intId));
-};
+const getBreakoutUser = user => Users.find({
+  extId: new RegExp(user.userId),
+  connectionStatus: 'online',
+}).fetch();
+
+const verifyUserThatJoinedOnBreakout = userArray => userArray.length === 1;
+
+const filterUsersNotAssigned = filter => users => users.filter(filter);
+
+const mapUsersToNotAssined = mapFunction => users => users.map(mapFunction);
+
+const flatUsersArray = usersArray => usersArray.flat();
+
+const getUsersNotAssigned = _.pipe(
```

Comment:
Consider adding a `// `comment - in order to obtain the list of viewers who have not entered a breakout room, we need to filter the list of viewers ...